packages/amplify-category-analytics/src/provider-utils/awscloudformation/cloudformation-templates/pinpoint-cloudformation-template.json

```diff
@@ -233,10 +265,13 @@
           }
         },
         "Handler": "index.handler",
-        "Runtime": "nodejs14.x",
+        "Runtime": "nodejs12.x",
```

Comment:
What's the reason for making this node12?

Reply:
not sure how that got in there. reverted